library/alloc/src/vec/mod.rs

```diff
@@ -152,12 +152,13 @@ mod spec_extend;
 /// assert_eq!(vec, [7, 1, 2, 3]);
 /// ```
 ///
-/// The [`vec!`] macro is provided to make initialization more convenient:
+/// The [`vec!`] macro is provided for convenient initialization:
```

Comment:
The changed wording feels weird although it is shorter.

Reply:
Similarly to above, avoiding "more" avoids raising the question "relative to what?". It also moves the sentence away from passive voice and towards active voice; going all the way would result in "The `vec!` macro provides convenient initialization" (which perhaps I should have done).